packages/block/src/header.ts

```diff
@@ -187,7 +196,8 @@ export class BlockHeader {
     extraData: Buffer,
     mixHash: Buffer,
     nonce: Buffer,
-    options: BlockOptions = {}
+    options: BlockOptions = {},
+    baseFeePerGas?: BN
```

Comment:
I find this a bit inconsistent/confusing that `baseFeePerGas` is after the opts. I understand trying to minimize breaking changes though. Perhaps in the next round of breaking changes we can update this to `constructor(blockData, opts)` like the tx package so we can continue to add/remove/modify block fields while maintaining a consistent order of params.